features/Collection.feature

```diff
@@ -12,30 +12,14 @@ Feature: Collection Commands
       | "the-hive-th"     | { "city": "changmai", "district": 7 } |
     And I refresh the collection
     # collection:export
-    When I run the command "collection:export" with:
+    When I run the command "export:collection" with:
       | arg  | nyc-open-data |                            |
       | arg  | yellow-taxi   |                            |
       | flag | --query       | { term: { city: "hcmc" } } |
       | flag | --scrollTTL   | 3s                         |
     Then I successfully call the route "collection":"delete" with args:
       | index      | "nyc-open-data" |
       | collection | "yellow-taxi"   |
-    # collection:import
-    And I run the command "collection:import" with args:
```

Comment:
Can you keep this test and use the `import` command instead to validate importing a collection is still working?